app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.scss

```diff
@@ -1,6 +1,39 @@
 @import "../pb_button/button";
 @import "../pb_button/button_mixins";
 @import "../tokens/colors";
+@import "../tokens/spacing";
+
+
+@mixin horizontal_border($opacity: $opacity, $border_color: true) {
+  @if $border_color == true {
+    background-color:rgba($white, $opacity);
+  } @else {
+    background-color:rgba($bg_dark, $opacity);
+  }
+  content: '';
+  position: relative;
+  width: calc(100% + #{$space_md} + #{$space_md});
+  height: 2px;
+  top: calc(-#{$space_xs} - 1px);
+  bottom: 0;
+  left: -$space_md;
+  display: block;
+}
+
+@mixin vertical_border($opacity: $opacity, $border_color: true) {
+  @if $border_color == true {
+    background-color:rgba($white, $opacity);
+  } @else {
+    background-color:rgba($bg_dark, $opacity);
+  }
+  content: '';
+  position: relative;
+  width: 2px;
+  height: $space_md;
+  top: 0;
+  bottom: 0;
+  right: calc(#{$space_lg} + 2px);
+}
```

Comment:
These could possibly be combined into 1 mixin

Reply:
Another thought is moved to a separate mixin file to make this a bit more readable